packages/utils/src/lib/reports/generate-stdout-summary.ts

```diff
@@ -17,11 +17,13 @@ function addLine(line = ''): string {
 }
 
 export function generateStdoutSummary(report: ScoredReport): string {
+  const { categories = [] } = report;
+  const printCategories = categories.length > 0 ? true : false;
```

Comment:
The `categories` aren't optional in `Report` (only in `CoreConfig`). And `<boolean expr> ? true : false` is redundant. So this can be simplified:

```suggestion
  const printCategories = report.categories.length > 0;
```

Reply:
Simplified in both generate files.